### DISCUSSION TITLE
Similar commit hashes detected when creating tags

### DISCUSSION THREAD

**Discussion by matt-ryley-Quant** [tj-actions/changed-files#1393] [Q&A] [answered]
Hi there!

I am having some issues using `tj-actions/changed-files` and was wondering if you could help at all. 
Our workflow is set-up such that when we create a new tag of the main branch, the workflow and the corresponding jobs are triggered. One of these jobs uses `tjactions/changed-files`. 

This job fails with the following error message

`Error: Similar commit hashes detected: previous sha: 7589eb52caa278b36e31b414906c382df93ff49e is equivalent to the current sha: 7589eb52caa278b36e31b414906c382df93ff49e.`

We tag using the GitHub web browser which I believe doesn't create a new commit for the tag so this error does make some sort of sense. Namely the actions triggered by tagging have the same commit hash as the last commit to main. (At least from my current understanding, please correct me if I am wrong :) )

I have also tried using the GitHub CLI to create and push annotated tags, but that resulted in the same error. Which should in theory create a new commit for the tag.

My question is two-fold.

1) Is this expected behaviour when using `tj-actions/changed-files` in a workflow triggered on creation of a new tag?
2) If this is the expected behaviour, could you advise on how to use `tjactions/changed-files` for workflows triggered by creation of new tags?

Thank you!

**Comment by jackton1** [comment, marked as answer]
Hi @matt-ryley-Quant, you'll have to provide some more information by creating an Issue [here](https://github.com/tj-actions/changed-files/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc). Please include your workflow file with the checkout action and tj-actions/changed-files portions, if possible re-run the workflow with debug enabled and attach the logs.

You can also take a look at an example workflow that runs on either on release created event or tag push [here](https://github.com/tj-actions/demo3/blob/main/.github/workflows/release.yml).

**Reply by matt-ryley-Quant** [reply]
Thanks for the reply, I will get more information to you when I can :)

**Comment by jackton1** [comment]
Closing this pending further information.
